### ISSUE TITLE
Crash when clicking links within "About" section of settings

### ISSUE THREAD

**Issue by JamesHooper0820** [MoneyFox/MoneyFox#2210]
**Describe the bug**
Clicking on any of the links within the "About" section of Settings results in Money Fox crashing, and the link destination not opening.

**To Reproduce**
1. Click on the settings cog on the top right of Money Fox.
2. Click on the "About" tab.
3. Click on any of the links within this area, such as "www.github.com".

**Expected behavior**
A new tab is opened in the user's browser taking them to that link destination, and no crash occurs.

**Screenshots**
Occurs on any clickable link within this area.
![image](https://user-images.githubusercontent.com/46788598/153312966-a1b8641a-d362-4d91-9621-91dfdeeff08f.png)

**Desktop (please complete the following information):**
 - OS: Windows, Build 19042.1466
 - Version of Money Fox: Latest as of 10/02/2022

**Additional context**
Only tested on Windows 10.

**Comment by NPadrutt** [comment]
Thanks for reporting this @JamesHooper0820 
Do you want to have a look at this and submit a PR?

**Comment by JamesHooper0820** [comment]
> Thanks for reporting this @JamesHooper0820 Do you want to have a look at this and submit a PR?

Would be more than happy to - if possible it'd be useful if you could point me in the rough location you'd believe this to be in since I'm not familiar with the structure of the codebase. Thanks

**Comment by kg8z** [comment]
Hi @JamesHooper0820  and @NPadrutt ,

I'm new to open source contribution so please forgive any breaches of etiquette or silly questions...

The issue appears to be related to how the browser process is invoked in BrowserAdapter.cs. While Microsoft documentation suggests you can simply pass a URI into Process.Start, it appears you need to pass a ProcessStartInfo object with UseShellExecute = true for this to work. Perhaps previous versions of the .NET framework behaved differently.

Anyhow, I tested the following and it worked properly:

            var browserProcStartInfo = new ProcessStartInfo(uri.ToString());
            browserProcStartInfo.UseShellExecute = true;
            Process.Start(browserProcStartInfo);

As mentioned, I'm new to this project, open source in general, and GIT (I'm an old Subversion guy). I'd be happy to create a branch and submit a PR with these changes if you want. Just know it will be my first time doing so in a public GitHub project.

**Comment by NPadrutt** [comment]
hey @kg8z 
yeah, there was a change when I migrated from UWP to WinUI. Probably I made an error there.
If that is fine with @JamesHooper0820 I would say go ahead and submit a PR, given that you already have a tested solution for this.

**Comment by JamesHooper0820** [comment]
Hi @NPadrutt, I've got a fix I'd like to push to master but I'm having some trouble with "permissions denied". Tried googling amongst other things but not sure how to get around this.

**Comment by NPadrutt** [comment]
hey @JamesHooper0820
Ah, it sounds like you cloned the repo directly. You would have to fork it instead via the fork button in the top right: 
<img width="105" alt="image" src="https://user-images.githubusercontent.com/1764367/154811838-02c9355f-c159-40ad-b7fa-6c82b8b5c5d5.png">

That basically create a connected copy of the repo that belongs to you. When you clone that you can create a branch, make the adjustments and push that branch. Once you've done that you can create a Pull Request in this repo here. When you go to the Pull Request section here it should already suggest to you to open one if you have a connected branch with changes.

So in short:
1. Fork this Repo
2. Clone your fork
3. Create a feature branch and make the changes
4. Push that branch to your fork
5. Create a pull request here

Once that is done it will automatically trigger a validation pipeline that will make sure all tests are running etc. and I can review it and then hopefully merge it.

I hope that helps! :)

**Comment by JamesHooper0820** [comment]
Thank you very much I'll do that now :)
